### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by salvis2**
Setting up structures for documentation updates. Hopefully I will offer initial looks at every how-to and topic listed in this initial commit. Looking to work more on this in the coming week.
@yuvipanda what is your procedure for testing sphinx? And how do you deploy the documentation?

### CODE REVIEW

**Comment by salvis2** [comment]
So I am running `sphinx-build -b html . built-docs` to test the documentation. Only warning I get right now is 

```
WARNING: html_static_path entry '_static' does not exist
```

So I'm fairly confident that these are looking good syntax-wise. Still more docs to come.

@yuvipanda, I've noticed that you manually line break your `.rst` files; is this to prevent large diffs in git? Should I also do this?

**Comment by yuvipanda** [comment]
@salvis2 this is awesome!

Yes, I manually line break my .rst files for diffs. Would love if you could do that too! I think my editor has helpers for it as well.

Let's merge this right after!

**Comment by salvis2** [comment]
Okay, I'll look at line breaks in the morning and try to make it my next commit. Do you have a character limit set for that?

**Comment by salvis2** [comment]
I've realized I don't actually know a lot about the image building (I know what command to run, but that's about it), so I was going to leave `docs/howto/hubploy-build-jupyterhub-image.rst` for someone else to fill in. The file is linked in `docs/howto/index.rst` at the moment, but if you want it unlinked until there is content, that is an easy change.

File: docs/howto/hubploy-deploy-jupyterhub-repo-setup.rst [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+You also need the following tools installed:
+
+#. Your cloud vendor's commandline tool.
+
+   #. `Google Cloud SDK <https://cloud.google.com/sdk/>`_ for Google Cloud
+   #. `AWS CLI <https://aws.amazon.com/cli/>`_ for AWS
+   #. `Azure CLI <https://docs.microsoft.com/en-us/cli/azure/>`_ for Azure
+
+#. A local install of `helm 3 <https://helm.sh/docs/intro/install/>`_. Helm 2 is also supported, 
+   but requires the same version of Helm to be present locally and on the cluster. If you are sing
```
Comment (consideRatio) [inline docs/howto/hubploy-deploy-jupyterhub-repo-setup.rst:38]:
```suggestion
   but requires the same version of Helm to be present locally and on the cluster. If you are using
```

**Comment by yuvipanda** [comment]
Thanks for working on this, @salvis2! \o/

**Pull request merged**